commons/phone/phone_contacts_list.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.phone",
+  "type": "record",
+  "name": "PhoneContactsList",
+  "doc": "Number of new contacts added/removed from the list of contacts between this and the previous record.",
```

Comment:
Either we make this type more specific, like `PhoneContactListModification` or we add a more generic description here.

Reply:
Added a better description